lib/internal/process/promises.js

```diff
@@ -41,9 +50,12 @@ function setupPromises(scheduleMicrotasks) {
       var reason = pendingUnhandledRejections.shift();
       if (hasBeenNotifiedProperty.get(promise) === false) {
         hasBeenNotifiedProperty.set(promise, true);
+        const uid = promiseToGuidProperty.get(promise);
         if (!process.emit('unhandledRejection', reason, promise)) {
-          // Nobody is listening.
-          // TODO(petkaantonov) Take some default action, see #830
+          process.emitWarning('Possibly unhandled promise rejection ' 
```

Comment:
I'm happy with `emitWarning` here — the value in double emitting is that, as a user of a system that is designed around NDJSON-based logs, I can easily hook into the warning system and turn this message into a JSON log (vs. having to rely on patching out `console`.) 

I'd quibble that we should probably log this as an `Unhandled promise rejection`, versus "possibly unhandled promise rejection." At the moment of logging, the promise _is_ unhandled, and we cannot say whether or not it will be handled in the future. If it is possible for the rejection to become handled, that case will be caught by the "rejection handled" warning.